### DISCUSSION TITLE
Github Pages not working at all

### DISCUSSION THREAD

**Discussion by suchomelb** [community/community#42936] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I went through a portfolio tutorial to make a web page for my portfolio to use github pages.  Of course, in the video describing it, everything works, but on my screen, nope.  No web page.

https://github.com/suchomelb/BryanSuchomel.github.io

Don't know what I'm missing.

**Comment by SilasRodrigues19** [comment]
This is your page link:

[https://suchomelb.github.io/BryanSuchomel.github.io/](https://suchomelb.github.io/BryanSuchomel.github.io/)

You can go to [this link](https://github.com/suchomelb/BryanSuchomel.github.io/deployments) and click on "View Deployment"
